src/coreComponents/finiteElement/elementFormulations/FiniteElementBase.hpp

```diff
@@ -79,18 +82,117 @@ class FiniteElementBase
    */
   virtual ~FiniteElementBase() = default;
 
+  /**
+   * @struct StackVariables
+   * @brief Kernel variables allocated on the stack.
+   *
+   * Contains variables that will be allocated on the stack. Used only by Virtual Element classes to
+   * hold the computed projections of basis functions
+   */
+  struct StackVariables
+  {
+    /**
+     * Default constructor
+     */
+    GEOSX_HOST_DEVICE
+    StackVariables()
+    {}
+  };
+
+  /**
+   * @struct Initialization
+   * @brief Variables used to initialize the class.
+   */
+  struct Initialization
+  {
+    /**
+     * Constructor
+     */
+    GEOSX_HOST_DEVICE
```

Comment:
Is this flag needed on the constructor? Do you plan to instantiate this object in a kernel? If not you can probably remove it